app/actions/remote/command.ts

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {IntlShape} from 'react-intl';
+import {Alert} from 'react-native';
+
+import {switchToChannel} from '@actions/local/channel';
+import {showPermalink} from '@actions/local/permalink';
+import {Client} from '@client/rest';
+import ClientError from '@client/rest/error';
+import {SYSTEM_IDENTIFIERS} from '@constants/database';
+import DeepLinkTypes from '@constants/deep_linking';
+import DatabaseManager from '@database/manager';
+import NetworkManager from '@init/network_manager';
+import {queryChannelsById} from '@queries/servers/channel';
+import {queryConfig, queryCurrentTeamId} from '@queries/servers/system';
+import {queryUsersByUsername} from '@queries/servers/user';
+import {showModal} from '@screens/navigation';
+import * as DraftUtils from '@utils/draft';
+import {matchDeepLink, tryOpenURL} from '@utils/url';
+
+import {makeDirectChannel, switchToChannelByName} from './channel';
+
+import type {DeepLinkChannel, DeepLinkPermalink, DeepLinkDM, DeepLinkGM, DeepLinkPlugin} from '@typings/launch';
+
+export const executeCommand = async (serverUrl: string, intl: IntlShape, message: string, channelId: string, rootId?: string) => {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        return {error: `${serverUrl} database not found`};
+    }
+
+    let client: Client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error: error as ClientError};
+    }
+
+    // const config = await queryConfig(operator.database)
+    // if (config.FeatureFlagAppsEnabled) {
+    //     const parser = new AppCommandParser(serverUrl, intl, channelId, rootId);
+    //     if (parser.isAppCommand(msg)) {
+    //         return executeAppCommand(serverUrl, intl, parser);
+    //     }
+    // }
```

Comment:
do we need this comment here?

Reply:
This handle the Apps commands, which should be included in a different PR with all the apps stuff. We could keep it here for when we implement the Apps functionality, or remove it completely and salvage it from V1 when we get into that. Both work for me. (Same for the next comment)